File: examples/src/example_readings_aspnet_web_app/Models/WeatherReportRow.cs
using System;

namespace ArenaExamples.Readings.Aspnet.Models;

public class WeatherReportRow
{
    public long Id { get; set; }
    public DateTime RecordedAt { get; set; }
    public double Precipitation { get; set; }
    public double Humidity { get; set; }
    public double Pressure { get; set; }
}

File: examples/src/example_readings_spring_boot_web_app/src/main/java/arena/examples/readings/springboot/WeatherReportRow.java
package arena.examples.readings.springboot;

import com.fasterxml.jackson.annotation.JsonProperty;
import java.time.Instant;

public record WeatherReportRow(
    long id,
    @JsonProperty("recorded_at") Instant recordedAt,
    double precipitation,
    double humidity,
    double pressure) {}
